Sources/Misc/Obsoletions.swift

```diff
@@ -526,6 +526,48 @@ public extension Purchases {
         fatalError()
     }
 
+    /**
+     * Configures an instance of the Purchases SDK with a custom `UserDefaults`.
+     *
+     * Use this constructor if you want to
+     * sync status across a shared container, such as between a host app and an extension. The instance of the
+     * Purchases SDK will be set as a singleton.
+     * You should access the singleton instance using ``Purchases/shared``
+     *
+     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
+     *
+     * - Parameter appUserID: The unique app user id for this user. This user id will allow users to share their
+     * purchases and subscriptions across devices. Pass `nil` or an empty string if you want ``Purchases``
+     * to generate this for you.
+     *
+     * - Parameter observerMode: Set this to `true` if you have your own IAP implementation and want to use only
+     * RevenueCat's backend. Default is `false`.
+     *
+     * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
+     *
+     * - Warning: This assumes your IAP implementation uses StoreKit 1.
+     * - Warning: If you're using observer mode with StoreKit 2, ensure that you call ``Purchases/handleObserverModeTransaction(_:)`` after making a purchase.
```

Comment:
The second warning is kind of a no-op since this API is obsoleted, right? Meaning you can't use it in any case, so you would still have to move to the SK2 version of it? 

Assuming that's correct, maybe we can tell folks to move to that method as well, like
```suggestion
     * - Warning: This assumes your IAP implementation uses StoreKit 1.
     * - Warning: If you're using observer mode with StoreKit 2, configure the SDK with `configure(withAPIKey:appUserID:observerMode:storeKitVersion:)` passing in `.storeKit2` as the `storeKitVersion` and ensure that you call ``Purchases/handleObserverModeTransaction(_:)`` after making a purchase.
```